### ISSUE TITLE
new device request: OnePlus 7T/HD1907

### ISSUE THREAD

**Issue by Pokey1Kenobi** [AXP-OS/issue-tracker#10]
# Required information

- Device vendor and model ->OnePlus HD1907
- Device codename (if known) -> Hotdogb
- LineageOS device tree -> https://wiki.lineageos.org/devices/hotdogb/
- Supported by DivestOS already? -> Yes
- I am willing to provide logs (requires either Matrix or Telegram) -> Yes
# Note

_Currently not supported devices might get added but this won't work without your help. That means you have to admit to test new builds and maybe provide logs if something is not working as expected._

_(*)your request will likely get rejected if there is no official LOS reference_

**Comment by steadfasterX** [comment]
Device request state:

- [ ] update [AXP.OS manifest](https://github.com/AXP-OS/manifest)
- [ ] setup [vendor/firmware](https://code.binbash.rocks:8443/AXP.OS/vendor_firmware)
- [ ] setup [automation repo](https://github.com/sfX-android/automation_scripts/tree/ansible/roles/axp)
- [ ] setup semaphore
- [ ] compile
- [ ] if device is NOT supported by DOS: see "additional steps"
- [ ] test all AXP.OS [core features](https://github.com/AXP-OS/build/wiki/Features)
- [ ] if applicable: test bootloader re-lock
- [ ] add device to [the AXP.OS wiki](https://github.com/AXP-OS/build/wiki)
- [ ] prepare DL+OTA server
- [ ] provide a public beta version
- [ ] update website
- [ ] announce on XDA

additional steps (if device is NOT supported by DOS):

- [ ] setup kernel patching:
   - [ ] [cve-patcher](https://github.com/sfX-android/automation_scripts/tree/ansible/roles/axp/common/files/cve_fixup)
   - [ ] [defconfig](https://github.com/sfX-android/automation_scripts/blob/ansible/roles/axp/common/files/extend_dos/getDefconfig.patch)
   - [ ] fix boot issues by overwriting default CONFIGs (e.g. like [this](https://github.com/sfX-android/automation_scripts/blob/d20f7dce494c72e956eb4e7efde2c0705cb969b3/roles/axp/common/templates/divested.vars.j5y17lte.j2#L13C8-L13C28))

**Comment by Pokey1Kenobi** [comment]
Absolutely. But I'm going to need to wait a few days as I'm in the process of moving to a new city and this phone is my DD that I need to BU and I'm not currently proficient with this A/B device.

**Comment by steadfasterX** [comment]
> Absolutely. But I'm going to need to wait a few days as I'm in the process of moving to a new city and this phone is my DD that I need to BU and I'm not currently proficient with this A/B device.

no worries, do whenever you can and for sure ensure that you have a full backup of everything before starting! :) good luck with the movement ;)

**Comment by steadfasterX** [comment]
hey @Pokey1Kenobi any chance to test already?

**Comment by steadfasterX** [comment]
hey @Pokey1Kenobi any chance that you are able to test or should I close this request?

**Comment by steadfasterX** [comment]
I will close this for now. feel free to re-open at any time. The test build remains there so if you find time let me know

**Comment by Pokey1Kenobi** [comment]
Oh Hi. Glad we're still speaking.. 

I'm sure the installation is indeed straight forward. -I've had e.os and LOS on this 7t previously then some hiccups with losing fingerprint data and it not working let me back to O2 .  A head trauma is to blame(TBI/Coma from car accident) as it affects my concentration if I'm not very familiar with something so that has some of most of the reason for not flashing. Long time from writing donor IMEIs on evo's to put them on boost for cheap service.
    I'm on the east coast, I've visited tg and looked around but not a fan and have used matrix so I'll have another look or try to get your attention there. 

Hey thanks a lot for keeping the door open.

**Comment by steadfasterX** [comment]
Hi @Pokey1Kenobi oh ofc we still speaking. I just thought you will have no time atm so I've closed it. Afaiu you still want to try? just let me know if you want to proceed or not so I (or you) can re-open that issue at any time again. I have used the 7T pro as my DD for long time and as they have the same specs I think this shouldn't be an issue but yea I cannot test it on my own before. that device is supported by LOS and DOS since a long time so I do not expect any issues but again.. who knows. without a full backup I won't risk flashing anything anyways. and if you have one you can go back at any time

**Comment by Pokey1Kenobi** [comment]
[image: 7.jpg]
Yes, I am still interested as before but my phone had an unfortunate fall
directly on its screen, again.
 Even with an Otter like case with builtin screen protector, it was over
quick.. So, I'm going to order another screen and fix it again, I suppose.
So bear with me for another week or two and go from there.

**Comment by steadfasterX** [comment]
hi  @Pokey1Kenobi any news? planning the august security patch builds rn and wanting to know if I should include hotdogb for this as well (only makes sense if you still plan to test ofc).

**Comment by Pokey1Kenobi** [comment]
hey there, yes, I'm currently waiting for my replacement lcd screen in the
mail before I'm able to load the rom.
I've been looking deeper into all your work and contributions to the
industry and want to share that I'm impressed by your dedication as well as
fortunate to have your willingness to assist me. Ironically, Divests Dev's
work ethic and dedication seems like the perfect inspiration. I hope I get
to see the New World Order of privacy in these spaces become reality and
you two are recognized
Been interested in, what I recall as being referred to as fuck widows use
linux a few years back. More appropriate I'd say but certainly not pc in
the bigger picture you deserve to be a part of.. I attempted to utilize
Boxes within Pop-OS to install mAid because my full disk encryption seemed
to be a burden but no love in the end. Currently going to add an old laptop
spinner to my tower to put mAid on, any advices?
Thanks again for all you do/done, and thinking about applying the HDb
security patch.
My apologies for not being up to speed for getting my 7T on board, a lots
been going with life lately..
Cheers, PWK

**Comment by steadfasterX** [comment]
all good! just wanted to know if I should keep it up or not ;) take ur time its not urgent. 

thanks for ur words ;) yea I cant count the days, weeks, months I spent on work for Open Source projects. Gladly I am able to do that because it's also (at least one) part of my RL.

Forget/F** Windows Use Linux (FWUL) was a bit to harsh for many ;) but I think "mAid" was a good replacement for the name.. what do u mean by HDb security patch? 
anyways mAid related stuff should go via the mAid [discussion options](https://code.binbash.rocks:8443/mAid/build/wiki#support) instead of here ;) other than that, well anything non-mainstream stuff like a exoctic graphic cards etc can cause issues when installing Linux (not just mAid) but well everything Manjaro supports is recommended for mAid as well

**Comment by Pokey1Kenobi** [comment]
HotDogb.

**Comment by steadfasterX** [comment]
if anyone wants to see this coming and can help with testing: reopen the issue and/or comment on it.

**Comment by Pokey1Kenobi** [comment]
My 7t is repaired with new screen finally.
    About to put your mAid into a partition on my PC.  Will be flashing axp
see you later tomorrow...

**Comment by Pokey1Kenobi** [comment]
- [ ] current ASB patch - System (`Android Settings -> Info/About -> Android Version` must be current)
- [ ] current ASB patch - Kernel (`Android Settings -> Info/About -> Kernel version -> build date` must be current. Kernel version must have `dos.pXXX` inside the version string)
- [ ] SELinux enforced (`Android Settings -> Privacy -> Trust`)
- [ ] Encryption enforced (`Android Settings -> Privacy -> Trust`)
- [ ] pre-rooted (`Magisk App -> "Magisk" must be shown as "installed"`)
- [ ] signature spoofing support (see Enable [Google Support](https://github.com/AXP-OS/build/wiki/Setup#optional-activate-google-support))
- [ ] microG included (`microG Settings -> Selftest -> ensure all checkboxes are ticked`)
- [ ] current [Google Play Store](https://github.com/AXP-OS/microg-phonesky-iap-support/releases) included
- [ ] Enable WireGuard VPN Kernel module (`Wireguard App -> grant root -> enable kernel support in settings`)
- [ ] OpenEUICC app included for devices supporting euicc ([Enable eSIM](https://divestos.org/pages/faq#activateEUICC))
- [ ] locking the bootloader on supported devices (see the device [wiki page](https://github.com/AXP-OS/build/wiki#installation-guides))
- [ ] Home Launcher Neo Launcher included
- [ ] extended list of F-Droid repositories (`F-Droid App -> Repositories -> find molly, nailyk, threema, futo_org`)
- [ ] AuroraStore App included
- [ ] boot debug log added (`adb shell ls -la [DEVICE-BOOT-DEBUG-PATH]/boot_debug` [DEVICE-BOOT-DEBUG-PATH] is defined in the [install guide](https://github.com/AXP-OS/build/wiki#installation-guides). Add the output in a comment)
- [ ] OTA update
- [ ] profit

**Comment by steadfasterX** [comment]
@Pokey1Kenobi :
Migrated to [gitea](https://code.binbash.rocks:8443/AXP.OS-public/issue-tracker/issues/10)
